homeassistant/components/recorder/migration.py

```diff
@@ -2313,11 +2307,10 @@ class StatesContextIDMigration(BaseRunTimeMigrationWithQuery):
 
     required_schema_version = CONTEXT_ID_AS_BINARY_SCHEMA_VERSION
     migration_id = "state_context_id_as_binary"
+    migration_version = 2
```

Comment:
Is this intentional?

Reply:
Nm, I see it in the opening text.
Maybe we should do that in a separate PR?

Reply:
OK, I'll revert the bumps from this PR.